src/test/java/com/systemmeltdown/robotlib/triggers/DriverSlowTriggerTest.java

```diff
@@ -0,0 +1,35 @@
+package com.systemmeltdown.robotlib.triggers;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class DriverSlowTriggerTest {
+	@Test
+	public void testGetReturnsTrueWhenHeld() {
+    XboxController controller = mock( XboxController.class );
+
+    when( controller.getTriggerAxis(Hand.kRight)).thenReturn(1.0);
+
+    DriverSlowTrigger trigger = new DriverSlowTrigger(controller);
+
+		assertEquals( trigger.get(), true );
+
+		trigger.close();
+	}
+
+	@Test
+	public void testGetReturnsFalseWhenOpen() {
+    XboxController controller = mock( XboxController.class );
+
+    when( controller.getTriggerAxis(Hand.kRight)).thenReturn(0.0);
+
+    DriverSlowTrigger trigger = new DriverSlowTrigger(controller);
+
+		assertEquals( trigger.get(), false );
+
+		trigger.close();
+	}
+}
```

Comment:
Maybe add a couple of other tests like what if it's exactly equal to the threshold or slight above or below.